### PR TITLE
multirotor landinging sudden fall protection:

### PR DESCRIPTION
**Pull request by tumbili**
remove condition which made activation of the protection
very unlikely

### CODE REVIEW

File: src/modules/mc_pos_control/mc_pos_control_main.cpp [hunk 1]
```diff
@@ -1517,7 +1517,6 @@ MulticopterPositionControl::task_main()
 
 						/* if we suddenly fall, reset landing logic and remove thrust limit */
 						if (hrt_elapsed_time(&_landing_started) > 15e5
-								&& _landing_thrust < FLT_EPSILON
 								/* XXX: magic value, assuming free fall above 4m/s2 acceleration */
 								&& (_acc_z_lp > 4.0f
```
Comment (tumbili) [inline src/modules/mc_pos_control/mc_pos_control_main.cpp]: @AndreasAntener @LorenzMeier We should discuss these magic values here. Might be worth it to have them on the safe side.

**Comment by LorenzMeier** [comment]
If we make the protection more likely to kick in, can we get the throttle lock back in as well? Without the throttle lock you might not fall, but you will flip on the ground instead. A successful landing is one without falling and without flip, one of them is not enough.

**Comment by AndreasAntener** [comment]
We should do a new test to check if the throttle lock is really necessary. Because initially I had no max tilt angle limiting during landing either. It's possible that the flipping is prevented by that already. Also, zero velocity is detected quickly on ground impact and thrust will then be set to 0.

BTW this condition is true on every landing as soon as it touches ground. The problem I still had in the end is that the magic values @tumbili is referring to are relatively strict and it released the thrust cut after touchdown sometimes.

**Comment by tumbili** [comment]
@AndreasAntener @LorenzMeier Sorry, it was my intention to put the thrust limiter back in. When I rebased on master the commit came back in. Should be correct now. Let's do some testing in simulation and in real to see what makes most sense.

**Comment by AndreasAntener** [comment]
The landing without the thrust limiting works as well as before, but without it falling out of the sky. Will add a new issue explaining 2 corner cases with the current approach.

**Comment by AndreasAntener** [comment]
This shows the landing when it works perfectly: http://logs.uaventure.com/view/LRF7FArYc3bZzVSGAT5wc5

**Comment by AndreasAntener** [comment]
Rebased and merged

**Pull request closed without merging**